backtester/eventhandlers/exchange/exchange.go

```diff
@@ -104,6 +104,26 @@ func (e *Exchange) ExecuteOrder(o order.Event, data data.Handler, bot *engine.En
 		limitReducedAmount = reducedAmount
 	}
 
+	// Conforms the amount to fall into the minimum size and maximum size limit after reduced
+	switch f.GetDirection() {
+	case gctorder.Buy:
+		if limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize {
+			f.SetDirection(common.CouldNotBuy)
+			e := fmt.Sprintf("Order size  %.8f exceed minimum size %.8f or maximum size %.8f ", limitReducedAmount, cs.BuySide.MinimumSize, cs.BuySide.MaximumSize)
+			f.AppendReason(e)
+			return f, fmt.Errorf(e)
+
+		}
+
+	case gctorder.Sell:
+		if limitReducedAmount < cs.SellSide.MinimumSize || limitReducedAmount > cs.SellSide.MaximumSize {
+			f.SetDirection(common.CouldNotSell)
+			e := fmt.Sprintf("Order size  %.8f exceed minimum size %.8f or maximum size %.8f ", limitReducedAmount, cs.BuySide.MinimumSize, cs.BuySide.MaximumSize)
```

Comment:
Are the variables used here supposed to be cs.SellSide instead of cs.Buyside? I've only done a quick review of this so far so I could be completely wrong

Reply:
@MadCozBadd Yes, you are right. A mistake. Thanks!